doc/api/cli.md

```diff
@@ -107,6 +107,20 @@ added: v0.11.14
 
 Throw errors for deprecations.
 
+### `--debug=<port>`
+<!-- YAML
+added: v6.2.0
+-->
+
+listen for debugger commands on user specified port. If port is not specified e.g `--debug` then use 5858 as default.
```

Comment:
Long line (80 columns max) and sentences should be properly capitalized.  The version number is not correct but I don't really remember when they were added.